doc/Programs/01-debugging.pod6

```diff
@@ -107,6 +79,43 @@ a string:
     #   in sub outer at /tmp/script.p6 line 3
     #   in block <unit> at /tmp/script.p6 line 4
 
+=head2 Environment variables
+
+See L<Raku Environment
+Variables|https://github.com/rakudo/rakudo/wiki/dev-env-vars> and
+L<Running rakudo from the command
+line|https://github.com/rakudo/rakudo/wiki/Running-rakudo-from-the-command-line>
+for more information.
+
+=head1 Ecosystem debugging modules
+
+There are at least two useful debuggers and two tracers available for Rakudo
+(the Raku compiler).  For more information on these modules, please refer to
+their documentation.
+
+Historically other modules have existed and others are likely to be
+written in the future. Please check the L<Raku
+Modules|https://modules.raku.org/> website for more modules like these.
+
+=head2 L<Debugger::UI::CommandLine|https://modules.raku.org/repo/Debugger::UI::CommandLine>
```

Comment:
Maybe surround it with the traditional C<>?

Reply:
Sounds good